### DISCUSSION TITLE
posts/thistle-tea/

### DISCUSSION THREAD

**Discussion by giscus** [pikdum/blog#3] [Comments]
# posts/thistle-tea/

https://pikdum.dev/posts/thistle-tea/

**Comment by nhh** [comment]
Awesome work! Will definitely do some digging around the code.

**Comment by ken-kost** [comment]
Amazing work, truly inspirational. 
P.S. What is this github comment section, how did you do that? I really like it.

**Reply by pikdum** [reply]
Thanks! The comment section uses https://giscus.app/.
